subjects/weareunique/README.md

```diff
@@ -0,0 +1,60 @@
+## Weareunique
+
+### Instructions
+
+Write a function that takes two strings and returns the number of characters that are not included in both, without doubles.
+- If there is no unique charachters return 0.
+- If both strings are empty  or one of them return -1.
```

Comment:
We should make this only when both are empty, when only one is empty it should return the unique chars...
so this: {"everyone", ""} and {"","everyone"} will return 6

Reply:
okay , it's done